services/resourcesCategory.js

```diff
@@ -13,7 +13,7 @@ const resourcesCategoryService = {
     return { count, items }
   },
   getResourcesCategoriesNames: async (match) => {
-    return await ResourcesCategory.find(match).distinct('name').exec()
+    return await ResourcesCategory.find(match).select('-author -createdAt -updatedAt').exec()
```

Comment:
.select('name')

Reply:
if we use select('name') it also send createdAt, updatedAt but we don`t need it here